integrationtest/neo/client/request/internal/GetAll.d

```diff
@@ -275,32 +275,11 @@ private scope class GetAllImpl
     {
         auto getall_status = cast(RequestStatusCode)status;
```

Comment:
This variable is a bit deceptive now, as there are no special GetAll status codes.